app/views/layouts/application.html.erb

```diff
@@ -24,47 +24,29 @@
 
 </head>
 
-<body>
-
+<body class="overflow-auto">
   <%= render 'layouts/navigation' %>
+
+
   <!--if login as Admin-->  
-  <% if logged_in? && current_user.admin? %>
-    <div class="panel-group">
-      <div class="panel panel-primary">
-        <div class="panel-heading">Welcome to Admin panel</div>
-        <div class="panel-body">
-          <div class="container">
-            <%= yield %>
-          </div>
-        </div>    
-      </div>         
-   </div>
-  <% end %>
-  <% if logged_in? && !current_user.admin? %>
-    <div class="panel-group">
-      <div class="panel panel-primary">
-        <div class="panel-heading">Welcome to MyShopee</div>
-        <div class="panel-body">
-          <div class="container">
-            <%= yield %>
-          </div>
-        </div>    
-      </div>         
-    </div>
+   
+  <% if logged_in? %>
+    <div class="container"  style="height: 550px;">
+      <div class="body"  ">   
+        <%= yield %>
+        <br> 
+        <br>
+      </div>
+    </div>    
   <% end %>
   <% if !logged_in? %>
```

Comment:
if not == unless

Reply:
why dont you use, 
if
else
end

block

instead of writing multiple if blocks